Cargo.toml

```diff
@@ -162,7 +162,9 @@ hyper-openssl = "0.9"
 http = "0"
 into_other = { path = "components/into_other", default-features = false }
 keys = { path = "components/keys", default-features = false }
-kvproto = { git = "https://github.com/pingcap/kvproto.git", default-features = false }
+# TODO(mornyx): Migrate when kvproto merges.
+# kvproto = { git = "https://github.com/pingcap/kvproto.git", default-features = false }
+kvproto = { git = "https://github.com/mornyx/kvproto.git", branch = "resource-metering", default-features = false }
```

Comment:
Ref to L261-L265, using patch then you don't need to replace kvproto dependencies everywhere. You can consider patch as `go mod replace` for rust.

Reply:
done.